slsDetectorSoftware/src/CmdProxy.cpp

```diff
@@ -893,22 +925,26 @@ std::string CmdProxy::DacValues(int action) {
         os << "\n\tGets the list of commands for every dac for this detector."
            << '\n';
     } else if (action == defs::GET_ACTION) {
-        std::vector<std::string> names = DacCommands();
-        std::vector<std::string> res(names.size());
-        std::vector<std::string> args;
-        for (size_t i = 0; i < names.size(); ++i) {
+        auto det_type = det->getDetectorType().squash(defs::GENERIC);
+        auto t = det->getDacList();
+        std::vector<std::string> res(t.size());
```

Comment:
Works but would be more elegant to to have to call through the command line interface. But I guess the CTB is the problem here.

Reply:
Yes, because if we use ToString for ctb as well, we will need to give the detector type as argument as well